step/step.go

```diff
@@ -54,15 +56,33 @@ func (r *ReportGenerator) ProcessConfig() (*Config, error) {
 	}
 
 	stepconf.Print(input)
-	r.logger.Println()
 	r.logger.EnableDebugLog(input.Verbose)
 
+	patterns := strings.Split(input.XcresultPatterns, "\n")
+	var filteredPatterns []string
+
+	for _, pattern := range patterns {
+		if pattern == "" {
+			continue
+		}
+
+		if !strings.HasSuffix(pattern, ".xcresult") {
```

Comment:
why not `filepath.Ext()` ? here we are validating the extension. Also this extension requirement is not mentioned in the input's description

Reply:
I intentionally did not use the `filepath.Ext()` because the name suggests that the patterns are file paths. But they are actually glob patterns which first need to be resolved to be actual valid file paths. For example the `/?a/[a,b]/[0-9]/**/*.xcresult` pattern might not be even a valid file path.